client/src/pages/admin/mentor-registry.tsx

```diff
@@ -79,172 +98,118 @@ function Page(props: Props) {
-                width: 200,
-                title: 'Github',
-                dataIndex: 'githubId',
-                sorter: stringSorter('githubId'),
-                render: (value: string, { name }) => {
-                  return (
-                    <>
-                      <GithubUserLink value={value} />
-                      <div>{name}</div>
-                    </>
-                  );
-                },
-                ...getColumnSearchProps(['githubId', 'name']),
-              },
-              {
-                title: 'City',
-                dataIndex: 'cityName',
-                width: 120,
-                sorter: stringSorter('cityName'),
-                ...getColumnSearchProps('cityName'),
-              },
-              {
-                title: 'Preferred',
-                dataIndex: 'preferedCourses',
-                render: (values: number[]) => tagsRenderer(values.map(v => courses.find(c => c.id === v)?.name ?? v)),
-              },
-              {
-                title: 'Pre-Selected',
-                dataIndex: 'preselectedCourses',
-                render: renderPreselectedCourses(courses),
-              },
-              {
-                title: 'Max students',
-                width: 80,
-                dataIndex: 'maxStudentsLimit',
-              },
-              {
-                title: 'Students Location',
-                width: 100,
-                dataIndex: 'preferedStudentsLocation',
-                sorter: stringSorter('githubId'),
-              },
-              {
-                title: 'Technical Mentoring',
-                dataIndex: 'technicalMentoring',
-                render: tagsRenderer,
-              },
-              {
-                title: 'Info',
-                dataIndex: 'info',
-                render: renderInfo,
-                width: 120,
-              },
-              {
-                title: 'Comment',
-                dataIndex: 'comment',
-              },
-              {
-                fixed: 'right',
-                width: 80,
-                title: 'Actions',
-                dataIndex: 'actions',
-                render: (_, record) => (
-                  <>
-                    <a onClick={() => setModalData(record)}>Edit</a> <br />
-                    <a onClick={() => cancelMentor(record.githubId)}>Cancel</a> <br />
-                    <a onClick={() => resendConfirmation(record)}>Re-send</a>
-                  </>
-                ),
-              },
-            ]}
-          />
-        </Col>
-      </Content>
-      {renderModal()}
-    </AdminPageLayout>
-  );
-}
+  const tabs = useMemo(() => {
+    const tabs = [
+      { key: MentorRegistryTabsMode.New, label: 'New applications', count: filterData(allData, false).length },
+      { key: MentorRegistryTabsMode.All, label: 'All Mentors', count: allData.length },
+    ];
+    return tabs.map(el => tabRenderer(el, activeTab));
+  }, [activeTab, data, filterData, allData]);
```

Comment:
`data` and `filterData` are unnecessary deps, please consider to exlude them from deps array